framework/source/class/qx/ui/form/Form.js

```diff
@@ -291,6 +291,26 @@ qx.Class.define("qx.ui.form.Form",
     },
 
 
+    /**
+     * Return an item by name.
+     *
+     * @param name {string} Item name.
+     * @return {qx.ui.form.IForm|null} The form item or null.
+     */
+    getItem : function(name) {
+      for (var i = 0; i < this.__groups.length; i++) {
+        var group = this.__groups[i];
+        for (var j = 0; j < group.names.length; j++) {
+          if (group.names[j] === name) {
+            return group.items[j];
+          }
+        }
+      }
+
+      return null;
```

Comment:
Hello.

I think will be nice throw an `Error` instead of only return null:

    if(qx.core.Environment.get("qx.debug")) {
        throw new Error("No field named " + name + " was found.");
    }

This can be useful for find typos:

    var myField = someForm.getItem('aTipo'); // is a typo :P